### DISCUSSION TITLE
[Feature] Progress bar for "Scan" job

### DISCUSSION THREAD

**Discussion by nZeus** [immich-app/immich#4384] [Feature Request]
### The feature

Hi,

I have connected a big external library and pressed the "Scan" button.
Currently, there is no way to see the status of the job.
Would be nice to know, if it's doing something, and how much is completed 🙏

Thank you

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by Purger-BB** [comment]
Just came here to ask for the same thing.
I'v added 400gb photo library,  would be great to know how far did it go after 8 hours 😁

**Reply by nZeus** [reply]
Mine is smaller, 150gb, and it took me a couple of days to process.
I also decided to split it into "smaller" libraries.
So, what I did:
- create a smaller library (e.g. by year),
- start scanning
- go to Administration -> Jobs
  - wait until all the jobs complete (you should see 0s everywhere). I also recommend to adjust the concurrency settings, especially the ones that had "1" I changed to at least 5.
- when the jobs complete, go and add the next folder/library...

**Reply by Purger-BB** [reply]
I should have done that, i actoualy pointed in my photo share and waiting for magic to happen

**Reply by nZeus** [reply]
Then you can check the Jobs page to see if something is still in progress.
Don't give up, the guys did a great job, I really enjoy the app, and performance is great in comparison to other similar apps 👍

**Reply by Purger-BB** [reply]
No giving up now, I'm amazed how this thing works! 
Just looked at jobs:
Thumbnails waiting 76000
Library waiting 178000
Tag objects waiting 28000
Faces 43000

Looks like my cpu gonna have a rough few days

**Comment by buha** [comment]
Yeah same request from me.

Running all this on Odroid-HC4 (Amlogic S905X3) so pretty slow compared to most people here.

What I did is that I disabled everything under Jobs except "GENERATE THUMBNAILS" and "LIBRARY". This way I speed this part up and I can already get a feeling of the progress by clicking randomly on the date scrollbar and see if there are thumbnails for that date. But this is just guessing, would be nice to know for good a percentage of the total files. 

Also, for me, the server stats shows wrong numbers, 71488 photos, 115 GB when I know for a fact it's more than 300 GB and the number of photos is much higher than 71488.

**Comment by bo0tzz** [comment]
Closing in favour of the more general #17352
